_includes/head/head.html

```diff
@@ -16,7 +16,6 @@
     https://github.com/CollectionBuilder/collectionbuilder-csv
 -->
 <meta name="generator" content="collectionbuilder-csv" />
-<meta http-equiv="Content-Language" content="{{ site.lang }}" >
```

Comment:
@koilebeit I am not sure if we should use the tag. I think we can safely remove it (see https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Language )

Reply:
I just saw https://github.com/Stadt-Geschichte-Basel/forschung.stadtgeschichtebasel.ch/blob/5a7c9e6fc274136dc0e0b1d4c1ad84d7ac8a654f/_layouts/default.html which solves this issue